bazel/conf/.bazelrc.build

```diff
@@ -120,6 +120,12 @@ build:afl --config=fuzzing
 build:afl --build_tag_filters=afl
 build:afl --run_under="/ic/bin/afl_wrapper.sh"
 
+# Fuzzing w/ Canister Sandbox configuration
+# NOTE: This is only for --config=fuzzing
+# AFL handles this differently in afl_wrapper.sh
+build:sandbox_fuzzing --config=fuzzing
+run:sandbox_fuzzing --run_under="ASAN_OPTIONS=detect_leaks=0:allow_user_segv_handler=1:handle_segv=1:handle_sigfpe=1:handle_sigill=0:quarantine_size_mb=16 LSAN_OPTIONS=handle_sigill=0 RUST_MIN_STACK=8192000"
```

Comment:
the doc does not explicitly say it works with env vars, but I guess you tried and it does?

https://bazel.build/reference/command-line-reference#flag--run_under

Reply:
Yep. I initially tried `action_env` but it only works during build time.